loc/ui/inventory.js

```diff
@@ -0,0 +1,7 @@
+export default {
+
+  inventory: {
```

Comment:
always use "name" when describing a particular module.
e.g. so it ends up being
``inventory.name`` 
rather than 
``inventory.inventory``

Reply:
literally? ### `name: {`